tests/lib/location/src/location_test.c

```diff
@@ -48,6 +48,10 @@ static const char ncellmeas_resp[] =
 	"%NCELLMEAS:0,\"00011B07\",\"26295\",\"00B7\",2300,7,63,31,"
 	"150344527,2300,8,60,29,0,2400,11,55,26,184\r\n";
 
+static const char ncellmeas_gci_resp[] =
+	"%NCELLMEAS:0,\"00023906\",\"24405\",\"5A00\",64,1830518,6400,116,53,16,1840265,1,0,"
+	"\"001E3904\",\"24405\",\"5A00\",65535,0,6400,42,48,7,1840265,0,0\r\n";
+
```

Comment:
Action for later, we need to test the max.

Reply:
Yep, should do that also for normal neighbors...